pyproject.toml

```
@@ -7,6 +7,7 @@ authors = [
     { name = "Elliott Kasoar" },
     { name = "Jacob Wilkins" },
     { name = "Alin M. Elena" },
+    { name = "Muhammad Mohsin" },
```

```suggestion
    { name = "Elliott Kasoar" },
    { name = "Jacob Wilkins" },
    { name = "Muhammad Mohsin" },
    { name = "Alin M. Elena" },
```
Generally we have Federica first, Alin last, and otherwise alphabetical